### DISCUSSION TITLE
Tasmota remota android apk

### DISCUSSION THREAD

**Discussion by isalopekx** [arendst/Tasmota#17704] [Q&A]
Hi to all.... Have one prpblem wuth the app. Im using premium apk Tasmota remota for android but it is only showing 6 sensors of sonoffth16 (5 temperatures and reley) and i have 7temp sensors and reley. Other apps are showing all sensors
Hepl please.
Regards

**Comment by sfromis** [comment]
Whatever app you are using there is not related to the Tasmota project. Better contact the creator.

**Reply by isalopekx** [reply]
Ok, tnx a lot.... I also tried to connect to mqtt web broker (hive and beebotte) bit have no luck. Any instruction how to do this? 
Tnx

**Reply by sfromis** [reply]
Check docs of the app you use, or the creator. The Tasmota project includes no apps. Requirements for connecting to a MQTT broker may be stricter when it is not the usual case of a broker running at home.

**Reply by isalopekx** [reply]
So no how to cennect help somwhere? Tnx

**Comment by barbudor** [comment]
I see a discussion on Pete Scargill's blog here : https://tech.scargill.net/just-a-quicky-tasmota-remota/
Where the author seems to answer (Rob Naylor) 
He also gave an email: tasmotaremota@orange.fr

**Reply by barbudor** [reply]
Now, for everything about connecting Tasmota to a Mqtt broker, everything is in Tasmota doc

Internet broker most likely requires TLS encryption which is not included in standard binaries for ESP8266 but is included in ESP32. So depending on the device you want to connect to hivehq or the other that I don't know, you may have to compile a variant by yourself

**Reply by isalopekx** [reply]
Yes, i see in information that is ESP8266... Sp no lucko for me to get it mqtt?

**Reply by barbudor** [reply]
I don't know. You don't expose your problem so we can't advise a solution
I already gave you a pointer: if your broker required TLS you need to self compile a binary with that option.
Read the docs

**Reply by sfromis** [reply]
While Sonoff TH 16 is indeed ESP8266, that has **nothing** to do with your troubles with that Android app. You have gotten a lot of MQTT setup working when you are getting 6 sensors shown. That means that Tasmota works, and the MQTT communication works. What the app is able to do with that is up to the creator of the app, and it is not part of the Tasmota project.

**Reply by isalopekx** [reply]
Hi...yes i know that....and that is no discussion here any more
Discusion is that i dont know how to setup mqtt to see tasmota sensors over internet...

**Reply by sfromis** [reply]
To quote you:
> Im using premium apk Tasmota remota for android but it is only showing 6 sensors of sonoffth16

I read that as you are able to get sensors shown, just not more than 6. Or did you not mean what you wrote....

**Reply by isalopekx** [reply]
Ejiyes...that was the first question, and i took your advice ande send developer an email. But after that i asked another (not to open another topic) about mqtt...

**Comment by isalopekx** [comment]
Hi....
i download your app , can  you please explain what to enter in app and
tasmota to get it working?
tnx

**Reply by barbudor** [reply]
You are still writing in that channel about the app while it is not sure that the app author will see it
Please address your question about the app to [tasmotaremota@orange.fr](mailto:tasmotaremota@orange.fr)

**Comment by isalopekx** [comment]
Ups....sry....
